src/main/java/seedu/address/model/person/Debt.java

```diff
@@ -11,15 +11,16 @@
  */
 public class Debt {
 
-    public static final String MESSAGE_DEBT_CONSTRAINTS =
-            "Debt can only contain numbers, and should have 1 or more digits";
-    public static final String DEBT_VALIDATION_REGEX = "\\d+";
+    public static final String MESSAGE_DEBT_CONSTRAINTS = "Debt must have at least 1 digit and be either "
+            + "a positive integer or a positive number with two decimal places";
```

Comment:
perhaps this can be simplified to something like "Debt must be a positive number with up to two decimal places"

Reply:
I think it would be better for debt to be either an integer or a number with 2 decimal places since those two representations are the most common. Debt field also cannot be empty. Hence, the requirement for at least 1 digit